config.toml

```
@@ -0,0 +1,333 @@
+default_branch = "rawhide"
+
+arch =  "x86_64" # used in sacks.py
+# arch is also used in sacks.py but technically can be different :/
+# Here we prefer the most reliable/fast architecture
+koji_arch = "x86_64" # used in bconds.py
+
+[deps]
+old = ["python(abi) = 3.11", "libpython3.11.so.1.0()(64bit)", "libpython3.11d.so.1.0()(64bit)"]
+new = ["python(abi) = 3.12", "libpython3.12.so.1.0()(64bit)", "libpython3.12d.so.1.0()(64bit)"]
+
+[components]
+excluded = ["python3.11", "python3.12"]
+extra = ["python3-docs"]
+
+[cache_dir]
+dnf = "_dnf_cache_dir"
+fedpkg = "_fedpkg_cache_dir"
+
+[multilib] # architectures to exclude in certain queries
+"x86_64" = "i686"
+
+[repos]
+[[repos.rawhide]]
+repoid = "rawhide"
+# metalink = "https://mirrors.fedoraproject.org/metalink?repo=rawhide&arch=$basearch"
+baseurl = ["http://kojipkgs.fedoraproject.org/repos/rawhide/latest/$basearch/"]
+
+[[repos.rawhide]]
+repoid = "rawhide-source"
+# metalink = "https://mirrors.fedoraproject.org/metalink?repo=rawhide-source&arch=$basearch"
+baseurl = ["http://kojipkgs.fedoraproject.org/repos/rawhide/latest/src/"]
+
+[[repos.target]]
+repoid = "python3.12"
+baseurl = ["https://copr-be.cloud.fedoraproject.org/results/@python/python3.12/fedora-rawhide-$basearch/"]
+#baseurl = ["http://kojipkgs.fedoraproject.org/repos/f39-python/latest/$basearch/"]
+metadata_expire = 60
+
+[bconds]
+[[bconds.python-setuptools]]
```

This should noo be a list of dicts, but a dict, no?

```suggestion
[bconds.python-setuptools]
```

---

Ignore this.